react-demo-app/src/main.tsx

```diff
@@ -0,0 +1,15 @@
+import ReactDOM from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
+import 'normalize.css'; // CSS reset
+import { store } from './store.ts';
+import { Provider } from 'react-redux';
+import { registerElements } from 'genesys-spark-components';
+
+
+registerElements();
+ReactDOM.createRoot(document.getElementById('root')!).render(
+  <Provider store={store}>
```

Comment:
I had to remove `React.StrictMode` here because when you're in a dev environment it renders everything twice for debugging purposes, which meant it was initializing twice.